plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions/VariationDescription.php

```diff
@@ -0,0 +1,72 @@
+<?php
+declare(strict_types=1);
+namespace Automattic\WooCommerce\Blocks\BlockTypes\AddToCartWithOptions;
+
+use Automattic\WooCommerce\Blocks\BlockTypes\AbstractBlock;
+use Automattic\WooCommerce\Blocks\BlockTypes\EnableBlockJsonAssetsTrait;
+
+/**
+ * VariationDescription class.
+ */
+class VariationDescription extends AbstractBlock {
+
+	use EnableBlockJsonAssetsTrait;
+
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'add-to-cart-with-options-variation-description';
+
+
+	/**
+	 * Render the block.
+	 *
+	 * @param array    $attributes Block attributes.
+	 * @param string   $content Block content.
+	 * @param WP_Block $block Block instance.
+	 *
+	 * @return string Rendered block output.
+	 */
+	protected function render( $attributes, $content, $block ) {
+		global $product;
+
+		if ( ! $product instanceof \WC_Product_Variable ) {
+			return '';
+		}
+
+		$variations                = $product->get_available_variations( 'objects' );
+		$formatted_variations_data = array();
+		foreach ( $variations as $variation ) {
+			$formatted_variations_data[ $variation->get_id() ] = array(
+				'variation_description' => wp_kses_post( (string) $variation->get_description() ),
+			);
+		}
+
+		wp_interactivity_state(
+			'woocommerce',
+			array(
+				'products' => array(
+					$product->get_id() => array(
+						'variations' => $formatted_variations_data,
+					),
+				),
+			)
+		);
+
+		$context = array(
+			'productElementKey' => 'variation_description',
+		);
+
+		$wrapper_attributes = array(
+			'data-wp-interactive'  => 'woocommerce/product-elements',
+			'data-wp-context'      => wp_json_encode( $context, JSON_NUMERIC_CHECK | JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_QUOT | JSON_HEX_AMP ),
+			'data-wp-bind--hidden' => '!state.productData.variation_description',
+			'aria-live'            => 'assertive',
```

Comment:
Does this need to be `assertive`?

[From boia.org](https://www.boia.org/blog/what-are-aria-live-regions):

> Normally, you’ll only use aria-live=”polite" unless you really need the user’s attention. For example, if their session is about to timeout, you might need to send an assertive announcement. 

This seems like a normal change, it shouldn't interrupt user-flow.

Reply:
It makes sense, thanks for noticing! Changed it in 57f0413b0b9822def67af8cb6fa01c9073daf8a6.